src/data/statuseffects.txt

```diff
@@ -2708,3 +2708,10 @@
 2708	On the Precipice of Carrots	maydaybar.gif	9b93c7a0436a4594d28bd8d03952cd13	neutral	none	eat 1 carrot cake precipice bar
 2709	Ready to Survive	maydaypackage.gif	e10348bbce52884781153e51f37ca337	neutral	none  use 1 MayDay&trade; supply package
 2710	Stuffed	maydaycan.gif	6e6cfdaf4191a47ecde914638c712803	neutral	none	eat 1 20-lb can of rice and beans
+2711	Ashamed	blush.gif	c9a095cc280c13e9ae6b44be77b61e79	neutral	none
+2712	Yapping Pal	dog.gif	dbdf338bfa4aba6ede9327f471914122	neutral	none
+2714	Earning Interest	dollarsign.gif	d02e47a3e379e6c280c66f226ffddc86	neutral	none	use 1 savings bond
```

Comment:
Interesting there is no effect #2713, yet. You might include a line with just that effect number on it, to indicate we know it is missing and not a typo.